Gems/Maestro/Code/Source/MaestroSystemComponent.h

```diff
@@ -53,8 +55,8 @@ namespace Maestro
 
     protected:
         ////////////////////////////////////////////////////////////////////////
-        // MaestroRequestBus interface implementation
-
+        // MovieSystemRequestBus interface implementation
+        IMovieSystem* GetMovieSystem() override { return m_movieSystem.get(); }
```

Comment:
Would it be easier if we register an AZ::Interface for the Movie System  (In `OnCrySystemInitialized`) instead of exposing this as an EBus? It'll make the changes in this file a little cleaner.